README.md

```diff
@@ -28,6 +22,18 @@ we recommend installing `casperlabs-client` in an isolated Python 3 environment
 or
 [virtualenv](https://virtualenv.pypa.io/en/latest/).
 
+These steps should help you set up your Python environment for the client.
+
+```
+sudo apt install gcc
+sudo apt install python3.7
+sudo apt-get install python3.7-dev
+virtualenv -p python3.7 env
+source env/bin/activate
```

Comment:
We probably need to make this clear that you need to run this every time you wish to activate the environment where you will run the client.